File: Apis/SharedDto/Dto/ProductDto.cs
namespace ManagementProducts.SharedDto.Dto
{
    public class ProductDto : AuditDto
    {
        public long? Id { get; set; } = null;
        public long CategoryId { get; set; } = 0;
        public string CategoryName { get; set; } = string.Empty;
        public string Code { get; set; } = string.Empty;
        public string Name { get; set; } = string.Empty;
        public decimal Price { get; set; } = 0;
        public string Unit { get; set; } = string.Empty;
        public decimal Stock { get; set; } = 0;
        public byte[]? Data { get; set; } = null;
        public string? Multimedia { get; set; } = null;
        public string? Extension { get; set; } = null;
        public string? Note { get; set; } = null;
        public decimal? FilterPriceMin { get; set; } = null;
        public decimal? FilterPriceMax { get; set; } = null;
        public decimal? FilterStockMin { get; set; } = null;
        public decimal? FilterStockMax { get; set; } = null;
        public bool HasMultimedia { get; set; } = false;
    }
}

File: www/src/app/usecase/dto/ProductDto.ts
export class ProductDto {
    Id: number = 0;
    CategoryId: number = 0;
    CategoryName: number = 0;
    Price: number = 0;
    Code: string = "";
    Name: string = "";
    Unit: string = "";
    Stock: string = "";
    Status: string = "";
    Note: string = "";
    HasMultimedia: boolean = false;

    FilterPriceMin: number = 0;
    FilterPriceMax: number = 0;
    FilterStockMin: number = 0;
    FilterStockMax: number = 0;
}